kolibri/content/api.py

```diff
@@ -57,7 +57,7 @@ def get_ancestor_topics(channel_id=None, content=None, **kwargs):
     :param content: ContentMetadata or str
     :return: QuerySet of ContentMetadata
     """
-    return content.get_ancestors().filter(kind="topic").using(channel_id)
+    return content.get_ancestors().filter(kind__kind="topic").using(channel_id)
```

Comment:
Sorry for the ambiguity of my question. I want to ask why capitalize the first letter in choices. Shouldn't them be the same value as in the constants?
```
choices = (
    (FACILITY, _("Facility")),
    (CLASSROOM, _("Classroom")),
    (LEARNERGROUP, _("LearnerGroup")),
)
```